README.md

```diff
@@ -15,10 +15,11 @@
 * [Deployment Pattern 1](advanced/am-pattern-1/README.md)
 * [Deployment Pattern 2](advanced/am-pattern-2/README.md)
 * [Deployment Pattern 3](advanced/am-pattern-3/README.md)
+* [Deployment Pattern 4](advanced/am-pattern-4/README.md)
 
 ## Changelog
 
-**Change log** from previous `v3.2.0.4` release: [View Here](https://github.com/wso2/kubernetes-apim/blob/3.2.x/CHANGELOG.md)
+**Change log** from previous `v3.2.0.4` release: [View Here](https://github.com/wso2/kubernetes-apim/blob/3.2.x/CHANGELOG.md)git
```

Comment:
Since we are already adding this to the release note let's remove this entry for now.

```suggestion
```